cmd/jag/commands/firmware.go

```diff
@@ -157,19 +158,46 @@ func BuildFirmwareEnvelope(ctx context.Context, id string, name string, wifiSSID
 		return nil, err
 	}
 
-	// TODO(kasper): Can we generate this in a nicer way?
-	wifiProperties := "{ \"wifi.ssid\": \"" + wifiSSID + "\", \"wifi.password\": \"" + wifiPassword + "\" }"
+	configMap := map[string]interface{}{
```

Comment:
It feels like we will have to start adding helpers for handling the assets (read from directory, etc.).